### ISSUE TITLE
Vita3k Android cresh

### ISSUE THREAD

**Issue by Aroxy4** [Vita3K/Vita3K-Android#152]
My device - redmi 9 prime
Mediatek - 80 prosser
Can't open any game 🙏 please fix it

**Comment by AMITYPREDATOR** [comment]
Yeah, same problem.  I'm using weaker hardware though. Samsung Galaxy A7 Lite streetfighter x tekken  shows the border screen with all the characters and then crashes to the dashboard.

**Comment by Aroxy4** [comment]
Yes just same prosser your divice and my device but what problem in my
Android and what game run in please tell

**Comment by AMITYPREDATOR** [comment]
Hey. I just downloaded  the new version.  Still crashing  before the game starts. Not trying to  hassle you, just keeping you informed about the issues.
